vimeo/VimeoNetworking
# AF-211: Adds compatibility with Swift 4

### Ticket
[AF-211](https://vimean.atlassian.net/browse/AF-211)

### Ticket Summary
Add support and compatibility with Swift 4.

### Implementation Summary
- Added `@objc` and `@nonobjc` attributes to Swift source files to designate Objective-C accessibility.
- Removed support for Swift 3 style Objective-C interface inference.
- Added an `account` property on **VIMLiveChatUser** (internal use only :wink:) to allow access to account type from Objective-C.
- Addressed a deprecation warning in **Request** regarding use of the `characters` property.
- Updated documentation.

### How to Test
- Run unit tests.
- Try integrating in a mixed-source project. Ensure expected APIs are available from Objective-C.
